atlp-rwanda/Team-Sostene-E-commerce-fe
# # Fix home and other components

#### What does this PR do?

- ensure that the home page looks appealing
- ensure that the contact page is responsive
- ensure that the seller items are not overlaping

#### Description of Task to be completed?

- Correct faults in some components

#### How should this be manually tested?

- Check the home page ('/' route)
- Check the contact page ('/contact' route)
- Check the seller items page ('/sellerItems' route)
